src/main/java/com/arturfrimu/exercisesback/service/ExerciseGenerationServiceImpl.java

```diff
@@ -270,17 +270,22 @@ public PercentageResponse getPercentage() {
                                 Collectors.toList())));
 
         BigDecimal totalPercentage = BigDecimal.valueOf(100);
+        BigDecimal correctPercentage = BigDecimal.ZERO;
+        BigDecimal errorPercentage = BigDecimal.ZERO;
+
+        if (percentage.containsKey(CORRECT) && !percentage.get(CORRECT).isEmpty()) {
```

Comment:
Verificarea a doua nu mai are sens in contextul dat.
Daca o stergem toate testele sunt oricum verzi

Sterge `!percentage.get(CORRECT).isEmpty()`